analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.CSharp8.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Tests.Diagnostics
+﻿using System;
+
+namespace Tests.Diagnostics
```

Comment:
I'm surprised you haven't fought your war against block-scoped namespaces for this file 😄

Reply:
Yes, in this case there are other namespaces, so I could not easily go for it :-)